TA Signals/TempMainView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  TempMainView.swift
+//  TA Signals
+//
+//  Created by Gabriel Maldonado Almendra on 27/7/21.
+//
+
+import SwiftUI
+
+struct TempMainView: View {
+    
+    // 1. The fetch property will observe the FetchToDo class for changes
+    @State var fetchedObject = StocksDataFetcher()
+    @State var showFetchDetails : Bool = false
```

Comment:
For these booleans you have here, you do not need to declare their type if you are setting them with a default value of false or true.  The compiler can figure out their type and set it for you.

```suggestion
    @State var showFetchDetails = false
```